packages/examples/stories/tests/placeholder.stories.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { withDesign } from 'storybook-addon-designs'
+
+export default {
+  title: 'Tests/Placeholder',
+  decorators: [withDesign]
+}
+
+export const showPlaceholder = () => (
+  <>
+    <pre>{`
```

Comment:
These buttons everywhere are for demonstrating real use-cases as much as possible. Please keep the file as it was (`.jsx` suffix is better, but it's not related to `paramKey` nor story source addon. I'll do batch change in future).

Reply:
the idea was to not have such a long story name in the menu, but ok